docs/src/pages/en/components/media-time-range.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: <media-time-range>
+description: Media Time Range
+layout: ../../../layouts/ComponentLayout.astro
+source: https://github.com/muxinc/media-chrome/tree/main/src/js/media-time-range.js
+---
+
+import ComponentSandpack from "../../../components/ComponentSandpack";
+
+The `<media-time-range>` component is used to indicate the current time position 
+in the timeline of the media, and provides controls to seek to a new time in the media.
+
+Hovering over the time range will show a preview time code by default, and if
+a track with a thumbnails VTT file is configured 
+a [preview thumbnail](#preview-thumbnails) will be shown.
+
+
+## Default usage
+
+<ComponentSandpack
+  client:load
+  height={250}
+  html={`<media-controller>
+  <video
+    playsinline muted crossorigin
+    slot="media"
+    src="https://stream.mux.com/A3VXy02VoUinw01pwyomEO3bHnG4P32xzV7u1j1FSzjNg/low.mp4"
+  ></video>
+  <media-control-bar>
+    <media-time-range></media-time-range>
+  </media-control-bar>
+</media-controller>`}
+/>
+
+
+
+## Preview thumbnails
+
+Adding the `<track default label="thumbnails" kind="metadata" src="">` 
+with a valid VTT file as `src` will enable the preview thumbnails on hover functionality.
```

Comment:
```suggestion
Adding a metadata text track labelled "thumbnails" (`<track default label="thumbnails" kind="metadata" src="">`) 
with a valid VTT file as `src` will enable the preview thumbnails on hover functionality.
```

Also, did we want to describe the format here? In another guide? Preview Thumbnails component?